openlibrary/plugins/openlibrary/js/availability.js

```diff
@@ -255,10 +255,10 @@ $(function(){
         })
         /* eslint-disable no-unused-vars */
         // event object is passed to this function
-        $('.searchResultItemCTA-lending form.join-waitlist').live('click', function(e) {
+        $(document).on('click', '.searchResultItemCTA-lending form.join-waitlist', function(e) {
```

Comment:
Can we add a docstring stating that this action triggers `join waitlist`?

Reply:
@mekarpeles should I add docstrings?